app/controllers/concerns/transaction_controllable.rb

```diff
@@ -24,7 +24,7 @@ def prep_transaction_form
     @loan_transaction_types = Accounting::Transaction.loan_transaction_type_options.select do |option|
       Accounting::Transaction::AVAILABLE_LOAN_TRANSACTION_TYPES.include?(option[1].to_sym)
     end
-    @qb_subtypes = [["Check", "Check"], ["None", nil]]
+    @qb_subtypes = Accounting::Transaction::QB_PAYMENT_TYPES.map { |t| [I18n.t("transactions.qb_object_subtype.#{t.downcase}"), t] }
```

Comment:
AVAILABLE_LOAN_TRANSACTION_TYPES are option sets in madeline. I used i18n instead, since these types are determined by QB and cannot be user created in any case.